targets/wasm_exec.js

```diff
@@ -283,12 +283,12 @@
 					},
 				},
 				gojs: {
-					// func ticks() float64
+					// func ticks() int64
 					"runtime.ticks": () => {
-						return timeOrigin + performance.now();
+						return BigInt((timeOrigin + performance.now()) * 1e6);
```

Comment:
Is `BigInt` performant enough for frequent calls to `runtime.ticks`? For example, is this line reasonably sure not to allocate?

Reply:
> Native support offered by BigInt often leads to better performance due to optimized operations built directly into the JavaScript engine.

I think it will be sufficient.

Reply:
If it turns out to be a performance issue, we can also do the conversion on the Go side. The code will just look a little bit uglier.